pkg/cache/cache_test.go

```diff
@@ -1502,8 +1507,16 @@ func TestFlavorInUse(t *testing.T) {
 				fooCq,
 				barCq,
 			},
-			wantInUse:                 true,
-			wantInUseClusterQueueName: fooCq.Name,
+			wantInUseClusterQueueNames: []string{fooCq.Name},
+		},
+		{
```

Comment:
Or you can directly remove the case above, I think the newly added case covered the scenario.

Reply:
Makes sense.

Reply:
Done.